app/web/features/communities/events/EventsPage.test.tsx

```diff
@@ -1,158 +1,24 @@
-import {
-  render,
-  screen,
-  waitFor,
-  waitForElementToBeRemoved,
-  within,
-} from "@testing-library/react";
+import { render, screen, waitFor } from "@testing-library/react";
 import userEvent from "@testing-library/user-event";
 import mockRouter from "next-router-mock";
 import { routeToNewEvent } from "routes";
-import { service } from "service";
-import events from "test/fixtures/events.json";
 import wrapper from "test/hookWrapper";
-import { assertErrorAlert, mockConsoleError, t } from "test/utils";
+import { t } from "test/utils";
 
 import EventsPage from "./EventsPage";
 
-const listAllEventsMock = service.events.listAllEvents as jest.MockedFunction<
-  typeof service.events.listAllEvents
->;
-
 describe("Events page", () => {
-  it("shows the 'Upcoming' tab by default", async () => {
-    listAllEventsMock.mockResolvedValue({
-      eventsList: events.filter((event) => !event.isCancelled),
-      nextPageToken: "",
-    });
-    render(<EventsPage />, { wrapper });
-
-    expect(
-      screen.getByRole("heading", {
-        name: t("communities:discover_events_title"),
-      })
-    ).toBeVisible();
-    expect(
-      screen.getByText(t("communities:discover_events_subtitle"))
-    ).toBeVisible();
-    expect(
-      await screen.findByRole("heading", { name: t("communities:upcoming") })
-    ).toBeVisible();
```

Comment:
The main tests are in `DiscoverEventsList` and `MyEventsList` now, only relevant one left here is the create new event button functionality.